```
Error: Missing @since tag for the "render_block_core_post_comments_link()" function.
Error: Missing @since tag for the "register_block_core_post_comments_link()" function.
Error: Missing @since tag for the "render_block_core_post_comments_count()" function.
Error: Missing @since tag for the "register_block_core_post_comments_count()" function.
Error: Process completed with exit code 1.
```
These are not introduced in this PR, but let me fix them.

--- a/packages/block-library/src/post-comments-count/index.php
+++ b/packages/block-library/src/post-comments-count/index.php
@@ -8,6 +8,8 @@
 /**
  * Renders the `core/post-comments-count` block on the server.
  *
+ * @since 6.9.0
+ *
  * @param array    $attributes Block attributes.
  * @param string   $content    Block default content.
  * @param WP_Block $block      Block instance.
@@ -33,6 +35,8 @@ function render_block_core_post_comments_count( $attributes, $content, $block ) 
 
 /**
  * Registers the `core/post-comments-count` block on the server.
+ *
+ * @since 6.9.0
  */
 function register_block_core_post_comments_count() {
 	register_block_type_from_metadata(

--- a/packages/block-library/src/post-comments-link/index.php
+++ b/packages/block-library/src/post-comments-link/index.php
@@ -8,6 +8,8 @@
 /**
  * Renders the `core/post-comments-link` block on the server.
  *
+ * @since 6.9.0
+ *
  * @param  array    $attributes Block attributes.
  * @param  string   $content    Block default content.
  * @param  WP_Block $block      Block instance.
@@ -59,6 +61,8 @@ function render_block_core_post_comments_link( $attributes, $content, $block ) {
 
 /**
  * Registers the `core/post-comments-link` block on the server.
+ *
+ * @since 6.9.0
  */
 function register_block_core_post_comments_link() {
 	register_block_type_from_metadata(

